static/css/zamboni/developers.css

```diff
@@ -85,6 +85,11 @@ ul.refinements:last-child {
     width: 370px;
 }
 
+form.border-width {
```

Comment:
`border-width` is a poor name for a class IMHO, it does not describe what it does or the element. Since this is fairly specific you could just use `#submit-describe` and not add the extra class.